styles/game/stage-brand-controls.css

```
@@ -0,0 +1,212 @@
+  }
+
+  body[data-theme="light"] .brand-mark[data-brand-grammar="jeo-parody"] .brand-bang {
+    color: #fff7d7;
+    text-shadow:
+      .03em .04em 0 #b87924,
+      0 0 .12em rgb(255 242 190 / .65);
+  }
+
+  .game-container [data-control-type] {
+    --control-skin-accent: var(--neon-yellow, #ffd84a);
+    position: relative;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)